environment.yml

```diff
@@ -2,7 +2,7 @@ name: pypeit
 channels:
   - defaults
 dependencies:
-  - python>=3.9,<3.12
+  - python>=3.10,<3.13
```

Comment:
I don't know how much is rilevant but give a look here: https://dev.to/hugovk/the-python-3-1-problem-85g

Have you test the changes?

Reply:
I played around with this using yaml.safe_load and it parses "python>=3.10,<3.13" as a single string, so it should work. But it wouldn't be a bad idea to wrap them in quotes for safety anyway.

Reply:
Thanks for pointing this out, @VascoSch92 , but, given @badpandabear 's comment, I think it's safe to leave it as is.